content/general/concepts/algorithm/terms/binary-search/binary-search.md

```diff
@@ -0,0 +1,70 @@
+---
+Title: 'Binary Search Algorithm'
+Description: 'An efficient searching algorithm'
```

Comment:
work on this a abit. We can try to improve this desc a bit more.

Reply:
@dakshdeepHERE I think Hisham updated the description. Should I change it or not?

Reply:
I did update the description

Reply:
Alright then its okay